contracts/utils/math/SafeCast.sol

```diff
@@ -116,7 +131,7 @@ library SafeCast {
      * - input must be greater than or equal to 0.
      */
     function toUint256(int256 value) internal pure returns (uint256) {
-        require(value >= 0, "SafeCast: value must be positive");
+        require(value >= int256(type(uint256).min), "SafeCast: value must be positive");
```

Comment:
I think this was better before. It's difficult to think about what the new expression means, and it's even relying in an unsafe Solidity cast.

```suggestion
        require(value >= 0, "SafeCast: value must be positive");
```